projects/plugins/jetpack/.gitignore

```diff
@@ -12,6 +12,7 @@
 /.cache/
 
 ## FILES
+/extensions/blocks/blocks-manifest.php
```

Comment:
Then this can go away too (the new file is already covered by line 5)
```suggestion
```

Reply:
Thanks, done in 17ae56bdcfb2